ocean/nemo/nemolite2d/psykal/Makefile

```diff
@@ -111,12 +111,12 @@ nemolite2d_acc: common_lib timer_lib serial_inf_lib nemolite2d_alg.f90 acc_trans
 nemolite2d_ocl: common_lib timer_lib ocl_inf_lib nemolite2d_alg.f90 ocl_transform.py
```

Comment:
Please extend comment to say we use `-I` to tell PSyclone where to find the module file `USE`d by a kernel.

Reply:
Done